### DISCUSSION TITLE
Configurable crotch illumination

### DISCUSSION THREAD

**Discussion by ThisIsBrutus** [ExpressLRS/ExpressLRS#1378] [Ideas]
Would it be possible to add a setting in either the lua or OLED menu, to change the color of the RGB manually? And turn it off completely?

It seems to be an option in the BetaFPV modules, at least with the firmware they shipped with.

**Comment by pkendall64** [comment]
I don't think we want to do that because we use the LED colors as status information and we sometimes ask "What is the RGB LED doing and what colour is it?".
If we allow custom colours then we can't use it as a diagnostic tool.

**Reply by ThisIsBrutus** [reply]
I was imagining that would stay as a mode for everyone who would want to keep it that way or need the information for troubleshooting. 

I don't own a BetaFPV module so I'm not sure of their implementation, but from a screenshot I can see that there is an "Auto" mode for RGB, which sounds like just that. 
Could also just be full on rainbow-mode, i'm not sure 😁
